src/services/placeService.js

```diff
@@ -17,7 +18,7 @@ class PlacesService {
 
       return response.data;
     } catch (e) {
-      console.log(e.message);
+      notificationsStore.addNotification({ message: 'Error fetching place data' + e.message });
```

Comment:
Please provide more specific message like - Error fetching autocomplete data. No need to post generic e.message onto error card as we dont know if this human readable or how big it might be. Better to add console.error log for this

Reply:
Will create and add to i18n